src/Components/Error/error.css

```diff
@@ -0,0 +1,70 @@
+
+*{
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+body{
+	background-color: whitesmoke;
+}
```

Comment:
You can delete this. Because it is already present in global stylesheet.

Reply:
Hmm got it.